src/win/conpty.cc

```diff
@@ -158,6 +160,12 @@ HRESULT CreateNamedPipesAndPseudoConsole(COORD size,
   return HRESULT_FROM_WIN32(GetLastError());
 }
 
+std::string from_wstring(const std::wstring &wstr) {
```

Comment:
move this to https://github.com/microsoft/node-pty/blob/main/src/win/path_util.cc